config.py

```diff
@@ -6,7 +6,11 @@
 
 # Define and set variables that are required in the config file.
 core_config_keys = [
+<<<<<<< HEAD
```

Comment:
These rebase conflict markers don't exist in my local branch. I don't know why they are here.